### DISCUSSION TITLE
move away from jQuery?

### DISCUSSION THREAD

**Discussion by trymeouteh** [fomantic/Fomantic-UI#1468] [General] [answered]
Will Formatic move away from jQuery? I know it is useful but I think it is better if Formatic had no dependencies and JavaScript has come a long ways, making it so jQuery is not needed for developing JS quickly.

Or even to have Formatic not require jQuery and have a optional jQuery integration JS script to allow people to use jQuery with Formatic if they wish.

**Comment by lubber-de** [comment, marked as answer]
Please see https://github.com/fomantic/Fomantic-UI/issues/374#issuecomment-496577211
jquery will be removed in v3
